libs/liquidity/src/lib/liquidity-table.tsx

```diff
@@ -35,21 +37,152 @@ export interface LiquidityTableProps
+      };
+
+      const stakeToCcyVolumeQuantumFormatter = ({
+        value,
+      }: ValueFormatterParams) => {
+        if (!value) return '-';
+        const newValue = new BigNumber(value)
+          .times(Number(stakeToCcyVolume) || 1)
+          .toString();
+        return `${addDecimalsFormatNumberQuantum(
+          newValue,
+          assetDecimalPlaces ?? 0,
+          quantum ?? 0
+        )}`;
+      };
+
+      const defs: ColDef[] = [
+        {
+          headerName: t('Party'),
+          field: 'party.id',
+          headerTooltip: t(
+            'The public key of the party making this commitment.'
+          ),
+        },
+        {
+          headerName: t(`Commitment (${symbol})`),
+          field: 'commitmentAmount',
+          type: 'rightAligned',
+          headerTooltip: t(
+            'The amount committed to the market by this liquidity provider.'
+          ),
+          valueFormatter: assetDecimalsQuantumFormatter,
+          tooltipValueGetter: assetDecimalsFormatter,
+        },
+        {
+          headerName: t(`Share`),
+          field: 'equityLikeShare',
+          type: 'rightAligned',
+          headerTooltip: t(
+            'The equity-like share of liquidity of the market used to determine allocation of LP fees. Calculated based on share of total liquidity, with a premium added for length of commitment.'
+          ),
+          valueFormatter: percentageFormatter,
+        },
+        {
+          headerName: t('Proposed fee'),
+          headerTooltip: t(
+            'The fee percentage (per trade) proposed by each liquidity provider.'
+          ),
+          field: 'fee',
+          type: 'rightAligned',
+          valueFormatter: percentageFormatter,
+        },
+        {
+          headerName: t('Market valuation at entry'),
+          field: 'averageEntryValuation',
+          type: 'rightAligned',
+          headerTooltip: t(
+            'The valuation of the market at the time the liquidity commitment was made. Commitments made at a lower valuation earlier in the lifetime of the market would be expected to have a higher equity-like share if the market has grown. If a commitment is amended, value will reflect the average of the market valuations across the lifetime of the commitment.'
+          ),
+          minWidth: 160,
+          valueFormatter: assetDecimalsQuantumFormatter,
+          tooltipValueGetter: assetDecimalsFormatter,
+        },
+        {
+          headerName: t('Obligation'),
+          field: 'commitmentAmount',
+          type: 'rightAligned',
+          headerTooltip: t(
+            `The liquidity provider's obligation to the market, calculated as the liquidity commitment amount multiplied by the value of the stake_to_ccy_siskas network parameter to convert into units of liquidity volume. The obligation can be met by a combination of LP orders and limit orders on the order book.`
```

Comment:
I believe it's no longer called stake_to_ccy_siskas.

Reply:
Yes, it is stakeToCcyVolume in the code. I'm updating the tooltip

Reply:
```suggestion
            `The liquidity provider's obligation to the market, calculated as the liquidity commitment amount multiplied by the value of the stake_to_ccy_volume network parameter to convert into units of liquidity volume. The obligation can be met by a combination of LP orders and limit orders on the order book.`
```